blueflood-core/src/test/java/com/rackspacecloud/blueflood/io/IOContainerTest.java

```diff
@@ -59,11 +61,28 @@ public void setup() throws Exception {
         // mock DatastaxIO.getSession() & Session
         PowerMockito.mockStatic( DatastaxIO.class );
         Session mockSession = mock( Session.class );
-        when( DatastaxIO.getSession()).thenReturn( mockSession );
+        when( DatastaxIO.getSession()).thenReturn(mockSession);
         PreparedStatement mockPreparedStatement = mock( PreparedStatement.class );
         when( mockSession.prepare( any( RegularStatement.class ) ) ).thenReturn( mockPreparedStatement );
-        when( mockSession.prepare( anyString() ) ).thenReturn( mockPreparedStatement );
-        when( mockPreparedStatement.setConsistencyLevel( any(ConsistencyLevel.class) ) ).thenReturn( mockPreparedStatement );
+        when( mockSession.prepare( anyString() ) ).thenReturn(mockPreparedStatement);
+        when( mockPreparedStatement.setConsistencyLevel(any(ConsistencyLevel.class)) ).thenReturn( mockPreparedStatement );
+    }
+
+
+    @Test
+    public void testDatastaxDriverConfig() {
```

Comment:
Can you put a TODO here , please, on why you have to move this up? I'll debug it the next time I have to touch this file.